src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -3,33 +3,37 @@
 <html>
 	<head>
 		<script>
-			async function gitpodMetricsAddCounter(metricsName, labels, value) {
-				function getMetricsUrl() {
-					const baseWorkspaceIDRegex = '(([a-f][0-9a-f]{7}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})|([0-9a-z]{2,16}-[0-9a-z]{2,16}-[0-9a-z]{8,11}))';
-					// this pattern matches URL prefixes of workspaces
-					const workspaceUrlPrefixRegex = RegExp(`^([0-9]{4,6}-)?${baseWorkspaceIDRegex}\\.`);
-					const url = new URL(window.location.href);
-					url.search = '';
-					url.hash = '';
-					url.pathname = '';
-					if (url.host.match(workspaceUrlPrefixRegex)) {
-						url.host = url.host.split('.').splice(2).join('.');
-					} else {
-						return;
-					}
-					const hostSegments = url.host.split('.');
-					if (hostSegments[0] !== 'ide') {
-						url.host = 'ide.' + url.host;
-					}
-					url.pathname = '/metrics-api';
-					return url.toString();
+			let workspaceId = null;
```

Comment:
@mustard-mh we should be careful with creating global variables, we had `gitpodMetricsAddCounter` on purpose to  avoid polluting global scope